pymaster/tests/test_field.py

```diff
@@ -42,6 +42,16 @@ def __init__(self):
 FT = FieldTester()
 
 
+def test_field_get_mask():
+    nside = 32
+    npix = hp.nside2npix(nside)
+    mp = np.random.randn(1, npix)
+    msk = np.random.rand(npix)
+    f = nmt.NmtField(msk, mp, n_iter=0)
```

Comment:
If the field is `lite`, does `get_mask` work, as well? (I guess yes, but just in case)

Reply:
Yes, fields always need a mask https://github.com/LSSTDESC/NaMaster/blob/8b95669a868ac75cd2a065677b6b4ab525781a37/src/nmt_field.c#L120

Reply:
Good. Thanks!